plugins/CoreVue/Commands/CompareHtmlTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CoreVue\Commands;
+
+use Piwik\Plugin\ConsoleCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CompareHtmlTest extends ConsoleCommand
```

Comment:
Did you commit that one on purpose? If so guess it should be available for development only.

Reply:
No, shouldn't have been committed.

Reply:
@sgiehl removed